tests/jerry/es2015/regression-test-issue-2544.js

```diff
@@ -13,4 +13,4 @@
 // limitations under the License.
 
 Object.defineProperty(Array.prototype, 0, { get : function () { throw $; } });
-Promise.race([ , this]).then(Error);
+Promise.race([ , this]).then(Error).catch(function(err) { assert(err instanceof ReferenceError); });
```

Comment:
Would it possible to test if the Promise executed at all? Otherwise it can happen that the test reports OK but the Promise was not invoked at all.

Reply:
There's a solution, I'll push it, feedback would be appreciated on it.